components/ryr-instructions.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+
+const InstTitleStyle = { textAlign: 'center', marginBottom: '0.8em', fontSize: '1.8em' };
+
+const InstTypes = { textAlign: 'center', marginBottom: '1.5em' };
+const InstTypes2 = { textAlign: 'center', marginBottom: '1.5em', textIndent: '-4.9em' };
+
```

Comment:
There is a lot of duplication between these objects, they should be built on top of each other (use `Object.assign`). This is also the perfect opportunity to start implementing styles with [emotion](https://emotion.sh/) as per our [conversion guide](https://github.com/request-yo-racks/web/wiki/Conversion-to-ReactJS#styling) 😉

Reply:
Okay